src/Generator/Passes/CleanCommentsPass.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using CppSharp.AST;
+
+namespace CppSharp.Passes
+{
+    public class CleanCommentsPass : TranslationUnitPass
+    { 
+        public override bool VisitDeclaration(Declaration decl)
+        {
+            if (decl.Comment != null)
+            {
+                var summaryPara = (ParagraphComment)decl.Comment.FullComment.Blocks[0];
```

Comment:
@ddobrev Are we guaranteed this will always be true? Shouldn't we be using an `as` here and testing for true? Or better yet, use a comment visitor?